packages/lit-dev-content/site/tutorials/content/first-component/06.md

```diff
@@ -0,0 +1,140 @@
+In this section we'll make it so when you click on the `word-viewer` the
+animation plays in reverse.
+
+Before adding the event handling we need a new piece of internal state to
+represent the current animation `playDirection`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  @state() private playDirection = 1;
+
+```
+
+```js
+// word-viewer.js
+
+  static properties = {
+    playDirection: {state: true},
+    ...
+  };
+
+  constructor() {
+    ...
+    this.playDirection = 1;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Next the math in `render()` to find the `word` needs to be updated to account
```

Comment:
Add comma and reword a bit because the passive voice sounds clunky

> Next, update the math that finds the displayed `word` in the `render()` function to account for...